ap.md

```diff
@@ -0,0 +1,228 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-8 columns">
+# Build Your Own Internet
+
+Along with connecting to an existing Wifi network, Tessel 2 can also act as a portable router and emit a custom Wifi network. We can use this functionality to connect directly with our Tessel through a web browser. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.  
```

Comment:
> We can use this functionality to connect directly with our Tessel through a web browser.

You might want to mention how it's different from connecting to your Tessel through a web browser when it's just connected to the same network as your host machine ie what are the benefits of Tessel hosting the wifi network?